src/python/aqueduct_executor/operators/connectors/data/extract.py

```diff
@@ -16,15 +18,71 @@ class RelationalParams(models.BaseParams):
     # any user-defined tags like `{{today}}`.
     query_is_usable: Optional[bool] = False
 
-    query: str
+    # exactly one of 'query' and 'queries' will be set.
+    # `query` represents a single query.
+    query: Optional[str] = None
+    # `queries` represents a chain of queries. We must first run _compile_chain
+    # to compile it to a single query before further processing.
+    queries: Optional[List[str]] = None
```

Comment:
nit: capitalize the first word in each comment

Reply:
I guess I meant for the "Exactly" comment. But yeah you're right, I think the capitalization should be consistent when we're using the backticks syntax.